gruntfile.js

```diff
@@ -17,6 +17,9 @@ module.exports = function( grunt ) {
 
 		jshint: {
 			options: {
+				globals: {
+					'CKEDITOR': false
```

Comment:
According to the Uglify docs:

> If value is false (default), JSHint will consider that variable as read-only.

As for the quotes, if you really made a question, just because it sounded like a list of "strings", not really properties of an object. :)